tests/Node.py

```diff
@@ -483,14 +484,21 @@ def waitForTransFinalization(self, transId, timeout=None):
         ret=Utils.waitForBool(lam, timeout)
         return ret
 
-    def waitForNextBlock(self, timeout=None, blockType=BlockType.head):
+    def waitForNextBlock(self, timeout=Timeout.default, blockType=BlockType.head):
         num=self.getBlockNum(blockType=blockType)
+        if isinstance(timeout, Timeout):
+            timeout = timeout.seconds(num, num+1)
         lam = lambda: self.getHeadBlockNum() > num
         ret=Utils.waitForBool(lam, timeout)
         return ret
 
-    def waitForBlock(self, blockNum, timeout=None, blockType=BlockType.head, reportInterval=None):
-        lam = lambda: self.getBlockNum(blockType=blockType) > blockNum
+    def waitForBlock(self, blockNum, timeout=None, blockType=BlockType.head, reportInterval=None, errorContext=None):
+#    def waitForBlock(self, blockNum, timeout=Timeout.default, blockType=BlockType.head, reportInterval=None, errorContext=None):
```

Comment:
Commented code.

Reply:
Commented code was what I wanted actually, so removed comment and replaced the executed line.